CHANGELOG.md

```diff
@@ -20,6 +20,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - The OpenEnclave CMake configuration now explicitly sets CMAKE_SKIP_RPATH to TRUE. This change should not affect fully static-linked enclaves.
 - oe_verify_attestation_certificate_with_evidence() has been deprecated because it has been deemed insufficient for security. Use the new, experimental oe_verify_attestation_certificate_with_evidence_v2() instead to generate a self-signed certificate for use in the TLS handshaking process.
 - In/out parameters in EDL now have the default count equals to one if the `count` attribute is not used.
+- The tool, `oecert`, has been renamed to `oegenerate`, as it generates not only certificates, but also reports and evidence in different formats.
```

Comment:
Don't we publish the tool starting from this release? I think the log should be under the Added section and be rephrased.  For example,

Publish a debugging tool `oegenerate`
- The tool, currently under the tools/ directory, was originally named `oecert` under the tests/tools directory.
- The tool can be used to generate certificates, reports, and evidence in various formats.
- The tool is for debugging purposes and is not suitable for production use.

Reply:
Sounds good, thanks!